tests/lib/rules/mapDispatchToProps-returns-object.js

```diff
@@ -93,3 +93,95 @@ ruleTester.run('mapDispatchToProps-returns-object', rule, {
     ],
   }],
 });
+
+ruleTester.run('mapDispatchToProps-returns-object-allowReturnBindFn', rule, {
+  valid: [
```

Comment:
Thank you so much for this. Would you please add some more negative cases. 
For example when the option is set to true but name of the function  is not `bindActionCreators ` and when the option is set to false.